spec/lib/services/find_admin_spec.rb

```diff
@@ -35,7 +35,7 @@
     end
 
     describe 'given an error occurs while retrieving the repository info' do
-      let(:error) { stub('error', :backtrace => [], :response => stub('reponse')) }
+      let(:error) { stub('error', :backtrace => [], :response => stub('response')) }
```

Comment:
I can't imagine this was correct...
I'm not sure what changing it will do (probably nothing material).